common/dal/src/context.rs

```diff
@@ -32,6 +32,8 @@ pub struct DalMetrics {
     pub write_rows: usize,
     /// Number of partitions scanned
```

Comment:
Number of partitions scanned, after pruning?

Reply:
:ok_hand: